homework.js

```diff
@@ -0,0 +1,61 @@
+//question 1 
+const summ = function(n) {
+    var sum = 0;
+    for (let i = 1; i<=n;i++)
+    sum += i ;
+    console.log(sum);
+    }
+    summ(5)
+VM562:5 15
+
+// question 2 
+let numbers = [ 2,4,6,8 ];
+
+let sum = numbers.reduce((a, b) => {
+    return a + b;
+});
+
+sum;
+20
+
+// question 3
+function summ () {
+for (i = 0; i<=)
+
+} 
```

Comment:
you can get the average of numbers by getting the sum then dividing by the count of the numbers